.github/workflows/pr-build-labeled.yml

```diff
@@ -46,6 +46,15 @@ jobs:
       - name: Checkout repo
         uses: actions/checkout@v4
       - name: build ${{ matrix.machine }}
-        uses: ./.github/actions/avocado-build
-        with:
-          machine: ${{ matrix.machine }}
+        env:
+          BB_GIT_SHALLOW: "1"
+          PARALLEL_MAKE: "-j16"
+          SDKMACHINE: "x86_64"
+          DISTRO_CODENAME: ""
+        run: |
+          echo "TARGET: avocado-distro"
+          echo "SDKMACHINE: x86_64"
+          echo "DISTRO_CODENAME: "
+
+          source ./scripts/init-build kas/machine/${{ matrix.machine }}.yml
+          PARALLEL_MAKE_ERLANG="-j16" kas build meta-avocado/kas/machine/${{ matrix.machine }}.yml --target avocado-distro
```

Comment:
`PARALLEL_MAKE` is being set in the env to the same value. We don't need to also set `PARALLEL_MAKE_ERLANG`